packages/constants/src/mcf.ts

```diff
@@ -177,6 +178,13 @@ export const SCHEMES: IScheme[] = [
     expiryDate: '2047-06-01',
     link: 'https://www.wsg.gov.sg/SGUnitedTraineeships-HostCompanies.html',
   },
+  {
+    id: SCHEME_ID.SG_UNITED_MID_CAREER_PATHWAYS,
+    scheme: 'SGUnited Mid-Career Pathways',
```

Comment:
> Display Text for new Scheme:
[  ] SGUnited Mid-Career Pathways Programme

Shouldn't be 'SGUnited Mid-Career Pathways Programme'?

Reply:
Not sure, the excel sheet given in the pivotal story gave the IcmsDescription as 'SGUnited Mid-Career Pathways'.

Reply:
Could you check w Ben?

Reply:
Clarified with Benjamin. To use 'SGUnited Mid-Career Pathways Programme' instead of 'SGUnited Mid-Career Pathways'.

Reply:
Renamed all, please review once more thanks!